am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -122,4 +126,36 @@ public FilterResourceResponse filterResource(String userId, String resourceId, J
 
         return null;
     }
+
+
+    /**
+     * <p>
+     * Method requires name of service, the type of resource, name of resource and a role name to
```

Comment:
It is not really Javadoc format I expect. Do not start with `method`. Check Sam's Javadoc for good example.

Reply:
👍